openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -96,21 +96,27 @@ func (r commonResult) Extract() (*Share, error) {
 	return s.Share, err
 }
 
-// CreateResult contains the result..
+// CreateResult contains the response body and error from a Create request.
 type CreateResult struct {
 	commonResult
 }
 
-// DeleteResult contains the delete results
+// DeleteResult contains the response body and error from a Delete request.
 type DeleteResult struct {
 	gophercloud.ErrResult
 }
 
-// GetResult contains the get result
+// GetResult contains the response body and error from a Get request.
 type GetResult struct {
 	commonResult
 }
 
+// GetExportLocationsResult contains the result body and error from an
+// GetExportLocations request.
+type GetExportLocationsResult struct {
+	commonResult
```

Comment:
This isn't a `commonResult`, is it? Does it return a `Share` in addition to an `ExportLocation`? If not, why should the method below stay `ExtractExportLocations` instead of `Extract`?